Makefile

```diff
@@ -587,6 +588,10 @@ release-alias-tag: ## Adds the tag to the last build tag.
 	gcloud container images add-tag $(KUBEADM_BOOTSTRAP_CONTROLLER_IMG):$(TAG) $(KUBEADM_BOOTSTRAP_CONTROLLER_IMG):$(RELEASE_ALIAS_TAG)
 	gcloud container images add-tag $(KUBEADM_CONTROL_PLANE_CONTROLLER_IMG):$(TAG) $(KUBEADM_CONTROL_PLANE_CONTROLLER_IMG):$(RELEASE_ALIAS_TAG)
 
+.PHONY: release-notes
+release-notes: $(RELEASE_NOTES)
+	go run ./hack/tools/release/notes.go --from=$(PREVIOUS_TAG) > release-notes-$(RELEASE_TAG).md
```

Comment:
Should we add these files into either a known folder like `out-release-notes/` and exclude them from gitignore? This would be useful in case we want to run the command locally

Reply:
added `_releasenotes/` and added to gitignore

didn't do it in this PR but we should maybe rename `out` to `_out`